README.md

```diff
@@ -1,6 +1,9 @@
 # Vet clinic database: create animals table
 > In this project I use a relational database to create the initial data structure for a vet clinic. I create a table to store animals' information, insert some data into it, and query it.
-
+## Before
+![Screenshoot-Before](./Before.png)
+## After
+![Screenshoot-After](./After.png)
```

Comment:
- [ ] I think you should have included a pair of screenshots with the results of the  EXPLAIN ANALYZE command for each of those queries (in total it should be 6 screenshots)

Reply:
Thank you for your suggestion.